Stratis.Bitcoin/Features/MemoryPool/MempoolBehavior.cs

```diff
@@ -13,30 +13,86 @@
 
 namespace Stratis.Bitcoin.Features.MemoryPool
 {
+    /// <summary>
+    /// Node behavior for memory pool.
+    /// </summary>
     public class MempoolBehavior : NodeBehavior
     {
-        // Average delay between trickled inventory transmissions in seconds.
-        //  Blocks and whitelisted receivers bypass this, outbound peers get half this delay. 
+        #region Fields
+
+        /// <summary>
+        /// Average delay between trickled inventory transmissions in seconds.
+        /// Blocks and whitelisted receivers bypass this, outbound peers get half this delay. 
+        /// </summary>
         const int INVENTORY_BROADCAST_INTERVAL = 5;
```

Comment:
Feel free to fix code style here (and below) as a part of this PR. Access modifier is missing here and PascalCase should be used.